package.json

```diff
@@ -48,5 +48,8 @@
       "main"
     ],
     "preset": "conventionalcommits"
+  },
+  "dependencies": {
+    "zod": "^3.19.1"
```

Comment:
wonder if this should be a peer dependency, since we are exposing the schemas directly? We wouldn't want consuming packages / apps to install a a different zod version that isn't compatible with the zod schemas written here..